projects/core/src/cart/facade/active-cart.service.ts

```diff
@@ -176,8 +176,6 @@ export class ActiveCartService implements OnDestroy {
 
   /**
    * Waits for the cart to be stable before returning the active cart's ID.
-   *
-   * @returns an observable with the active cart ID.
```

Comment:
guess we removed it since other methods in this service does not do it for JSdocs right?

Reply:
yeah, we agreed on Slack we'll be reducing the noise, and removing the `@` things in jsdocs.